resource/Resource.md

```diff
@@ -0,0 +1,137 @@
+# Resource API Overview
+The resource library primarily defines a type that captures information about the entity
+for which stats or traces are recorded. It further provides a framework for detection of
+resource information from the environment and progressive population as signals propagate
+from the core instrumentation library to a backend's exporter.
```

Comment:
Is there a difference between `core library` and `core instrumentation library`? Core library seems to be most consistently used throughout.

Is it worth giving an explicit definition of what the `core library` is? I assume it is any libraries provided under the Github `census-instrumentation` org which provide a common abstraction to backend exporters?

Reply:
I'd agree with the definition you gave, especially as exporters are moved to census-ecosystem.
What's the official take @bogdandrutu?

Reply:
I think core library is better.